src/components/loader-spinner/loader-spinner.spec.tsx

```diff
@@ -56,6 +56,20 @@ describe("LoaderSpinner", () => {
   });
 
   describe("when custom props are passed", () => {
+    it("when the 'spinnerLabel' prop is passed a custom string value it overrides the default visible label", () => {
+      render(<LoaderSpinner spinnerLabel="foo" />);
+      const visibleLabelElement = screen.getByTestId("visible-label");
+
+      expect(visibleLabelElement).toHaveTextContent("foo");
+    });
+
+    it("when the 'spinnerLabel' prop is passed a custom string value it overrides the visually hidden label", () => {
```

Comment:
nitpick(non-blocking): Same point here too. 

Something like: 
```suggestion
    it("should override the visually hidden label when the 'spinnerLabel' prop is passed a custom string value", () => {
```
or yet again something similar/better 😄